### ISSUE TITLE
Add linter for CSV data models (and lint before generating JSON-LD)

### ISSUE THREAD

**Issue by BrunoGrandePhD** [Sage-Bionetworks/schematic#546]
**Is your feature request related to a problem? Please describe.**

There are some cases that should be caught sooner rather than later. For example, double commas in a DependsOn list can result in weird behavior downstream. Also, as a user, I would like to know if a DependsOn value refers to an attribute that doesn't exist under Attribute (for example, because of inconsistent capitalization). 

**Describe the solution you'd like**

I would like to have a subcommand alongside `convert` that can lint a CSV data model and produce a list of errors and warnings. This linting should also be done before a JSON-LD data model is generated to warn the user of any potential issues. 

The command could look like this: 

```
schematic schema lint htan.model.csv
```

**How important is this feature?** Select from the options below:
• Medium - can do work without it; but it's important (e.g. to save time or for convenience)

**When will use cases depending on this become relevant?** Select from the options below:
N/A
